context/app/static/js/components/workspaces/NewWorkspaceDialog/NewWorkspaceDialog.tsx

```diff
@@ -0,0 +1,229 @@
+      onSubmit({
+        workspaceName,
+        templateKeys: [...selectedTemplates],
+        uuids: [...datasetUUIDs],
+      });
+    },
+    [datasetUUIDs, selectedTemplates, onSubmit],
+  );
+
+  return (
+    <>
+      <MenuItem
+        onClick={() => {
+          setDialogIsOpen(true);
+        }}
+      >
+        Launch Workspace
+      </MenuItem>
+      <Dialog
+        open={dialogIsOpen}
+        onClose={handleClose}
+        scroll="paper"
+        aria-labelledby="scroll-dialog-title"
+        aria-describedby="scroll-dialog-description"
+        maxWidth="lg"
+      >
+        <Box mb={2}>
+          <DialogTitle id="scroll-dialog-title" variant="h3">
+            Launch New Workspace
+          </DialogTitle>
+          <Box sx={{ px: 3 }}>
+            <Paper sx={{ p: 2 }}>
+              <Typography sx={{ mb: 2 }}>
+                Workspaces are currently Jupyter Notebooks that allows you to perform operations on HuBMAP data.
+              </Typography>
+              <Typography>
+                Three steps are shown for launching a workspace, but the only required field to launch a workspace is
+                “Step 2: Configure Workspace”. “Step 1: Edit Datasets Selection” is only required if there are issues
+                with any of the datasets selected, which will be indicated by an error banner.
+              </Typography>
+            </Paper>
+          </Box>
+        </Box>
+        <DialogContent dividers>
+          <Step title="Edit Datasets Selection" index={0}>
+            {children}
+            <Paper sx={{ p: 2 }}>
+              <Typography sx={{ mb: 2 }}>
+                To remove a dataset, select the dataset and press the delete button. If all datasets are removed, an
+                empty workspace will be launched.
+              </Typography>
+              <Typography>
+                To add more datasets to a workspace, you must navigate to the dataset search page, select datasets of
+                interests and follow steps to launch a workspace from the search page. As a reminder, once you navigate
+                away from this page, all selected datasets will be lost so take note of any HuBMAP IDs of interest, or
+                copy IDs to your clipboard by selecting datasets in the table below and pressing the copy button. You
+                can also save datasets to the “My Lists” feature.
+              </Typography>
+            </Paper>
+          </Step>
+          <Step title="Configure Workspace" isRequired index={1}>
+            <Box
+              id="create-workspace-form"
+              component="form"
+              sx={{
+                display: 'grid',
+                gap: 2,
+                marginTop: 1,
+              }}
+              // eslint-disable-next-line @typescript-eslint/no-misused-promises
+              onSubmit={handleSubmit(submit)}
+            >
+              <WorkspaceField
```

Comment:
Since the `value` prop is destructured from the props and `WorkspaceField` doesn't have type annotations, TypeScript treats `value` as a required property. Adding a default value of `undefined` fixes this; I can't make the suggestion on the `WorkspaceField` component since this PR didn't touch it, but here's what it would look like:
```
function WorkspaceField({ control, name, label, errors, value = undefined, ...rest }) {
```

Converting WorkspaceField to `.tsx` is another option.